envoy/upstream/host_description.h

```diff
@@ -202,6 +202,12 @@ class HostDescription {
    *         healthy state via an active healthchecking.
    */
   virtual absl::optional<MonotonicTime> lastHcPassTime() const PURE;
+  virtual MonotonicTime lastTrafficPassTime() const PURE;
+  virtual MonotonicTime lastTrafficPassTime2xx() const PURE;
+  virtual MonotonicTime lastTrafficPassTimeGrpc() const PURE;
+  virtual void setLastTrafficTime(MonotonicTime last_traffic_pass_time) const PURE;
```

Comment:
```suggestion
  virtual void setLastTrafficTimeTcpSuccess(MonotonicTime last_traffic_pass_time) const PURE;
```

? 

Methods above should be altered to match.